Plugins/BackbonePluginTemplatePlugin/BackbonePluginTemplatePluginGenerator.cs

```diff
@@ -131,6 +140,87 @@ private GlobalJsonModification CreateMsBuildSdk()
             return globalJsonModificationBuilder.Build();
         }
 
+        private Task AddProjectReferences()
+        {
+            var packagesReferences = CreateProjectModification();
+            return _projectManipulator.ManipulateProject(GetProjectFilePath(), packagesReferences);
+        }
+
+        private Task AddTestProjectReferences()
+        {
+            var testProjectModification = CreateTestProjectModification();
+            return _projectManipulator.ManipulateProject(GetTestProjectFilePath(), testProjectModification);
+        }
+
+        private ProjectModification CreateProjectModification()
+        {
+            var projectModificationBuilder = new ProjectModificationBuilder();
+            if (PluginVariant != VariantType.MinimalPlugin)
+            {
+                projectModificationBuilder = projectModificationBuilder.AddPackageReferences(CreateExtendedProjectPackageReferences());
+            }
+
+            if (UsesCentralPackageVersionsSdk)
+            {
+                projectModificationBuilder = projectModificationBuilder.AddSdk(CentralPackageVersions);
+            }
+
+            return projectModificationBuilder.Build();
+        }
+
+        private ProjectModification CreateTestProjectModification()
+        {
+            var projectModificationBuilder = new ProjectModificationBuilder();
+            projectModificationBuilder = projectModificationBuilder.AddPackageReferences(CreateTestProjectPackageReferences());
+
+            if (UsesCentralPackageVersionsSdk)
+            {
+                projectModificationBuilder = projectModificationBuilder.AddSdk(CentralPackageVersions);
+            }
+
+            return projectModificationBuilder.Build();
+        }
+
+        private static IEnumerable<PackageReference> CreateExtendedProjectPackageReferences()
+            => new[]
+            {
+                new PackageReferenceBuilder()
```

Comment:
I would put each of these package references into a constant.

Reply:
Addressed in: [Define constants for package references](https://github.com/messerli-informatik-ag/meta-generator/pull/27/commits/91aaeed0d5b3667b2e4a0bdfcaadc29ba3b44b4a)

Reply:
Addressed in: [Rename packages reference constants](https://github.com/messerli-informatik-ag/meta-generator/pull/27/commits/8a05813b7cf4ca79ecaf02f30ac9c3374ce4549e)